x-pack/plugins/security_solution/common/ecs/index.ts

```diff
@@ -63,4 +65,7 @@ export interface Ecs {
   // This should be temporary
   eql?: { parentId: string; sequenceNumber: string };
   Ransomware?: Ransomware;
+  // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Curious: did this really error? I see keys in other areas here starting with Capital letter and including underscores with no eslint disable rules on them

Reply:
`Memory_protection` fails to match this rule. `memory_protection` would work, but not with the upper M

Reply:
@paul-tavares can you point me to places where there are keys like this without eslint errors?